app/Files/TemporaryFile.php

```diff
@@ -0,0 +1,43 @@
+<?php
```

Comment:
there is this immediate upload thing i want to experiment needed tempory files  
how it works is , you drag and drop file into the area, then it immediately start upload as a tempory file, we have this temporary file thing because we want the file to be automatically deleted if user let say cancelled if you get what i mean. Only when user click on submit button, then this file is transfered into the actual storage.  then this temporary storage will always be purged automatically after each request.  There are many many advantages .